### ISSUE TITLE
[Plugin] [CopyFileHook] Copy collections of files

### ISSUE THREAD

**Issue by JusteRaimbault** [openmole/openmole#373]
Would it be possible to extend the CopyFileHook so that it take instead of a Val[File] a Val[any collection[File]], so that we can copy simultaneously different File variables to the same output dir (what is quite different from copying a given dir with all content recursively, files may be at different levels e.g.) ?

**Comment by JusteRaimbault** [comment]
Well, that was exactly what I was thinking about ; did not know the hook syntax could take a collection of hooks ! Thanks !
I test that and close if it work ! (although it may be good to wrap this and make it transparent for the noobs ? )

**Comment by jopasserat** [comment]
yes we could probably articulate it around a `MultiCopyFileHook` or `CopyFileMultiHooks` that would wrap the construction.
